plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -1500,12 +1503,14 @@ def get_snmp_destination_in_ccc(self, name):
                             return destination
                     time.sleep(1)
                 except Exception as e:
-                    if "Expecting value: line 1 column 1" in str(e):
-                        self.log(
-                            "Getting expection as SNMP destination with given name '{0}' not present in Cisco Catalyst"
-                            " Center.".format(name), "WARNING"
-                        )
-                        return None
+                    expected_exception_msg = ["Expecting value: line 1 column 1", "not iterable", "has no attribute"]
```

Comment:
same as above..

Can you please rephrase the log message?

Reply:
updated.